backend/tracim_backend/locale/de/LC_MESSAGES/tracim_backend.po

```diff
@@ -0,0 +1,387 @@
+# German translations for tracim_backend.
+# Copyright (C) 2021 ORGANIZATION
+# This file is distributed under the same license as the tracim_backend
+# project.
+#
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2021.
+# marie <marie.begue@algoo.fr>, 2021.
+msgid ""
+msgstr ""
+"Project-Id-Version: tracim_backend 3.7.1\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2021-05-03 12:31+0200\n"
+"PO-Revision-Date: 2021-04-30 16:42+0100\n"
+"Last-Translator: marie <marie.begue@algoo.fr>\n"
+"Language: de\n"
+"Language-Team: French <kde-i18n-doc@kde.org>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.6.0\n"
```

Comment:
Not really important but some value here doesn't look make much sense Language-Team, Report-Msgid-Bugs-To.
but also "ORGANIZATION" in the header.
We should probably add a small issue about configuring theses value properly and take some time just to understand the goal or not of these metadata.